speed-dreams/source-2.2.3/src/modules/userinterface/legacymenu/mainscreens/mainmenu.cpp

```diff
@@ -20,26 +20,26 @@
 #include <tgfclient.h>
 
 #include <racescreens.h>
-#include <playerconfig.h>
+#include <controlconfig.h>
 
 #include "mainmenu.h"
 #include "exitmenu.h"
 #include "optionsmenu.h"
 #include "creditsmenu.h"
 #include "DataSelectionMenu.h"
 
-
 static void *MenuHandle = 0;
 
-
+// SIMULATED DRIVING ASSISTANCE: renamed from onPlayerConfigMenuActivate
 static void
-onPlayerConfigMenuActivate(void * /* dummy */)
+onControlMenuActivate(void * /* dummy */)
```

Comment:
Currently the controls menu is accesible via the main menu and the racestart menu. It would probably be best to simplify this even more and to just remove this entirely from the main menu. Less path to take through the menu's is always better :)

Reply:
the racestart menu will probably be removed or drastically changed later on, when we have for example track selection in the research menu. Therefore it does not really matter at this point.